guides/bug_report_templates/action_controller.rb

```diff
@@ -7,17 +7,16 @@
 
   git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-  # Activate the gem you are reporting the issue against.
-  gem "rails", "~> 7.1.0"
+  gem "rails"
+  # If you want to test against edge Rails replace the previous line with this:
+  # gem "rails", github: "rails/rails", branch: "main"
 end
 
-require "rack/test"
 require "action_controller/railtie"
 
 class TestApp < Rails::Application
   config.root = __dir__
   config.hosts << "example.org"
-  config.session_store :cookie_store, key: "cookie_store_key"
```

Comment:
Unrelated but should we add `config.load_defaults`?

Reply:
:+1: A bunch of others templates had it added in #49631